tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -450,8 +450,16 @@ internal static class Telemetry
             public const string Enabled = "DD_INSTRUMENTATION_TELEMETRY_ENABLED";
 
             /// <summary>
-            /// Configuration key for the telemetry URL where the Tracer sends telemetry.
-            /// Ignored (and <see cref="ExporterSettings.AgentUri"/> is used instead) unless <see cref="ConfigurationKeys.ApiKey"/> is set.
+            /// Configuration key for sending telemetry direct to telemetry intake. If enabled, and
+            /// <see cref="ConfigurationKeys.ApiKey"/> is set, sends telemetry direct to intake. Otherwise, sends
+            /// telemetry to Agent. Enabled by default if <see cref="ConfigurationKeys.ApiKey"/> is available.
+            /// </summary>
+            public const string AgentlessEnabled = "DD_TRACE_TELEMETRY_AGENTLESS_ENABLED";
```

Comment:
I haven't found a place where this parameter is documented. Should we add it to a telemetry doc in GH or Confluence?